UI/installer/mp-installer.nsi

```diff
@@ -446,22 +464,25 @@ Section "un.obs-studio Program Files" UninstallSection1
 	RMDir "$INSTDIR\OBS Studio"
 SectionEnd
 
-Section /o "un.User Settings" UninstallSection2
+Section /o "un.Settings, Scenes, etc" UninstallSection2
```

Comment:
We already have users that end up having one of these issues:
1. "I reinstalled OBS but all my scenes are still there, why? I wanted a clean reinstall"
2. "I uninstalled OBS and now all my scenes are missing"

The goal of this change is to make it clearer for both situations.

The reason for why I didn't include the dot on the end is because the text as-is *just* fits, to the pixel, and a dot would likely add a horizontal scrollbar or be cut off.

Reply:
There already is a horizontal scrollbar for this on my screen because the first item is so long.
![image](https://user-images.githubusercontent.com/624931/82217643-b97c7f00-98e8-11ea-8c0c-37ba4aa7b22f.png)

Reply:
Updated the strings per your comments.